### ISSUE TITLE
Big Idea for a Big Update (Future)

### ISSUE THREAD

**Issue by nekokasumikancolle** [Monika-After-Story/MonikaModDev#4311]
I have two big idea to make this mod become more good.

First thing is
1) About the lemonade mod. i guess we need to make monika love it first. so we can add our affection. 
But first, let she made a trigger b4 summer. Like, "hey mc, do u know lemonade.." and she give fact about lemonade. And ask mc again about summer. so it can make a trigger to MC give lemonade. and later, monika said how about we make lemonade stand. and  we can make lemonade stand every summer. How about that

And second is
2) I guess its better if we have many random thing to do in special event. But we didnt have such as summer mode. And ik maybe it sound hard to do but. I guess its better if we have many random thing to do in special event. I mean like, in this winter We can play throw snow ball. And next year we drink hot chocolate together While read books. Next year we play make a snowman.
In conclusion is, Why we not make many activity in one season, and we make it randomize so we musnt have to update it too much in future. Like, rn is a big update, Cuz the idea have been pour it in this update 

I know maybe it sound too hard but if we can invite more artist and more mod editor. We can make this idea come true

**Comment by ThePotatoGuy** [comment]
1. lemonade stand seems out of nowhere, as in why would she set up a lemonade stand in the spaceroom
2. events are always planned things, having a specific event in mind is a better suggestion then mentioning several generic events.

[closing as rejected]
